### PR TITLE
add alias name for raft gpu index, milvus need.

### PR DESCRIPTION
**Pull request by Presburger**
/kind improvement

### CODE REVIEW

File: src/index/gpu_raft/gpu_raft_ivf_flat.cc [hunk 1]
```diff
@@ -30,4 +30,9 @@ KNOWHERE_REGISTER_GLOBAL(GPU_RAFT_IVF_FLAT, [](const int32_t& version, const Obj
                                                      cuda_concurrent_size);
 });
 
+KNOWHERE_REGISTER_GLOBAL(GPU_IVF_FLAT, [](const int32_t& version, const Object& object) {
+    return Index<IndexNodeThreadPoolWrapper>::Create(std::make_unique<GpuRaftIvfFlatIndexNode>(version, object),
+                                                     cuda_concurrent_size);
+});
+
```
Comment (liliu-z) [inline src/index/gpu_raft/gpu_raft_ivf_flat.cc:37]: Do we need to have both of this?

**Comment by liliu-z** [comment]
CP to 2.2 branch?

File: src/index/gpu_raft/gpu_raft_ivf_flat.cc (hunk 1 above)
Comment (Presburger) [reply src/index/gpu_raft/gpu_raft_ivf_flat.cc:37]: yes

**Pull request merged**